core/services/workflows/store/store.go

```diff
@@ -5,7 +5,8 @@ import (
 )
 
 type Store interface {
-	Add(ctx context.Context, state *WorkflowExecution) (WorkflowExecution, error)
+	Add(ctx context.Context, Steps map[string]*WorkflowExecutionStep,
```

Comment:
The rest of this file lowercases variable names :) Can we do that here too?

Reply:
oops, done, ta